changelog.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
-This file contains all the notable changes done to the Ballerina TCP package through the releases.
+This file contains all the notable changes done to the Ballerina OS package through the releases.
+
+## [Unreleased]
+
+### Added
+- [[2764] Add `setEnv()`, `unsetEnv()`, and `listEnv()` functions.](https://github.com/ballerina-platform/ballerina-standard-library/issues/2764)
```

Comment:
This format is a bit different from the other changelogs maintained by Stdlibs

https://github.com/ballerina-platform/module-ballerina-http/blob/master/changelog.md

Reply:
Updated in [3630028](https://github.com/ballerina-platform/module-ballerina-os/pull/423/commits/363002879574e551518cbdbfbe12e0a155333215)